tests/conditional_compilation/tools/infer_tool.py

```diff
@@ -74,6 +74,4 @@ def cli_parser():
     results = infer(ir_path=ir_path, device=device)
     np.savez(out_path, **results)
     log.info("Path for inference results: {}".format(out_path))
-    log.info("Inference results:")
-    log.info(results)
-    log.info("SUCCESS!")
```

Comment:
I'd rather control it with verbosity level. You can configure it to not print INFO level by default. You optionally then can add --verbose <LEVEL> into infer tool command line. Having that you can still debug infer results with the tool but have the logs of cc clean

Reply:
Le logger level configuration is on line #17.
https://github.com/openvinotoolkit/openvino/blob/385a7d762e6d9f02d3ad51866d71d91f5acf45af/tests/conditional_compilation/tools/infer_tool.py#L17

Reply:
Oh, that's good idea. Thanks :blush: